commit 79a553633eb16e59e99b8160e4b27adde3b8f8af
Author: Daniel Pocock <daniel@pocock.pro>
Date:   Thu Feb 24 20:56:54 2022 +0100

    FacadeRtpEndpointImpl: postConstructor: must call connectForwardSignals

diff --git a/src/server/implementation/objects/FacadeRtpEndpointImpl.cpp b/src/server/implementation/objects/FacadeRtpEndpointImpl.cpp
index 1b35257..134448a 100644
--- a/src/server/implementation/objects/FacadeRtpEndpointImpl.cpp
+++ b/src/server/implementation/objects/FacadeRtpEndpointImpl.cpp
@@ -152,6 +152,7 @@ FacadeRtpEndpointImpl::postConstructor ()
 
   rtp_ep->postConstructor();
   linkMediaElement(rtp_ep, rtp_ep);
+  connectForwardSignals();
 
 }
 
